expyfun/_experiment_controller.py

```diff
@@ -1931,11 +1933,10 @@ def _stamp_binary_id(self, id_, delay=0.03, wait_for_last=True):
         if not isinstance(id_, (list, tuple, np.ndarray)):
             raise TypeError('id must be array-like')
         id_ = np.array(id_)
-        if not np.all(np.logical_or(id_ == 1, id_ == 0)):
+        if not np.all((id_ == 1) | (id_ == 0)):
```

Comment:
```suggestion
        if not np.all(np.in1d(id_, [0, 1])):
```

Reply:
I think this is equivalent and maybe slightly clearer.  The current version might be carelessly misread as checking that `id_` is either all 1s or all 0s.